src/lib/features/frontend-api/frontend-api.e2e.test.ts

```diff
@@ -356,6 +356,12 @@ test('should store frontend api client metrics', async () => {
         projects: ['*'],
         environment: '*',
     });
+
+    // @ts-expect-error
```

Comment:
What's the expected error? It's usually a good idea to explain why. Also, ts-expect-error usually hides other potential errors. 
If it's a type issue with jest.fn, you can add type params to it: `jest.fn<string[]>` IIRC

Reply:
It's because `cachedFeatureNames` is not a publicly accessible property of `clientMetricsServiceV2`. I don't like adapting our code to make our tests happy (making this public instead of private when it doesn't make sense otherwise) but maybe that would be the easiest way of preventing this `// @ts-expect-error`. Thoughts?